modules/update-upgrading-web.adoc

```diff
@@ -68,4 +72,10 @@ ifdef::between[]
 --
 +
 You might need to perform several intermediate updates until you reach the version that you want.
-endif::between[]
+ifdef::rhel[]
++
+[NOTE]
+====
+When you update a cluster that contains Red Hat Enterprise Linux worker machines, those workers temporarily become unavailable during the update process. You must run the update playbooks on those machines for the cluster to finish updating.
```

Comment:
Maybe the approach 1) in https://bugzilla.redhat.com/show_bug.cgi?id=1786274#c28 is more accurate about this:  `You must run the upgrade playbook against each RHEL machine as it goes NotReady for the cluster to finish updating`.

Reply:
Lets go ahead with that, in parallel we'll work on removing this requirement. your suggested wording is fine with me.